src/components/AddAsset.tsx

```diff
@@ -56,8 +57,8 @@ export const AddAsset = ({ onClose }: { onClose: () => void }) => {
       );
       setLocalStatus(ActionStatus.SUCCESS);
     } catch (e) {
-      log.error({ title: e.message });
-      setErrorMessage(e.message);
+      log.error({ title: getErrorMessage(e) });
+      setErrorMessage(getErrorMessage(e));
       setLocalStatus(ActionStatus.ERROR);
```

Comment:
very very small, but probably best practice to cache the value of `getErrorMessage(error)` as a const in instances like this and below where the value isn't changing